languages/csharp/config.json

```diff
@@ -12,81 +12,56 @@
       {
         "slug": "basics",
         "uuid": "7d358894-4fbd-4c91-b49f-d68f1c5aa6bc",
-        "concepts": [
-          "variables-basic",
-          "methods-basic",
-          "integers-basic",
-          "comments-basic"
-        ],
+        "concepts": ["basics"],
         "prerequisites": []
       },
       {
         "slug": "strings",
         "uuid": "eda235a5-37de-4fb6-a5c5-cf7aa2309987",
-        "concepts": ["strings-basic"],
-        "prerequisites": ["methods-basic"]
+        "concepts": ["strings"],
+        "prerequisites": ["basics"]
       },
       {
         "slug": "numbers-floating-point",
         "uuid": "4eb8d86f-7123-473e-9d96-7939e9652608",
-        "concepts": ["numbers-floating-point", "loops-while"],
-        "prerequisites": [
-          "numbers-basic",
-          "type-conversion-numbers",
-          "conditionals-if"
-        ]
+        "concepts": ["floating-point-numbers", "while-loops"],
+        "prerequisites": ["basics", "numbers", "conditionals"]
       },
       {
         "slug": "numbers",
         "uuid": "93fbc7cf-3a7e-4450-ad22-e30129c36bb9",
-        "concepts": [
-          "numbers-basic",
-          "type-conversion-numbers",
-          "conditionals-if",
-          "methods-basic"
-        ],
-        "prerequisites": []
+        "concepts": ["numbers", "conditionals"],
+        "prerequisites": ["basics"]
       },
       {
         "slug": "enums",
         "uuid": "047b895b-7039-4af6-ad3a-17fd818d0b88",
-        "concepts": ["enums-basic", "conditionals-ternary"],
-        "prerequisites": [
-          "strings-basic",
-          "numbers-basic",
-          "generics",
-          "conditionals-if",
-          "type-conversion-numbers"
-        ]
+        "concepts": ["enums"],
+        "prerequisites": ["basics", "strings", "conditionals"]
       },
       {
         "slug": "dates",
         "uuid": "7762d9ec-ca95-4e10-b93c-afd2521787a2",
-        "concepts": ["dates-basic", "time-basic"],
-        "prerequisites": ["numbers-basic", "strings-basic", "classes-basic"]
+        "concepts": ["datetime"],
+        "prerequisites": ["basics", "numbers", "strings", "classes"]
       },
       {
         "slug": "enums-advanced",
         "uuid": "ad240550-d732-4125-b00b-b852027a2db7",
-        "concepts": ["enums-advanced"],
-        "prerequisites": [
-          "enums-basic",
-          "attributes-basic",
-          "bitwise-operations",
-          "numbers-integers"
-        ]
+        "concepts": ["enums-flags", "bitwise-operations"],
```

Comment:
I think this should be `enum-flags`

Reply:
It should actually be `flag-enums`, as that is how they are referred to. Thanks.